utils/checks.stanza

```diff
@@ -1166,7 +1178,7 @@ public pcb-check check-min-copper-copper-space (landpattern:LandPattern, side:Si
   val description = "Check min copper spacing"
   val category = "landpattern-check"
 
-  val copper = to-seq(copper(landpattern, side))
+  val copper = to-tuple(copper(landpattern, side))
```

Comment:
`to-seq` converts a `Seqable` to a `Seq`.
These objects are streams, and their elements are consumed / disappear after being read once.
The #CHECK condition consumes the stream by calling `min-space(copper)`.
The pass/fail messages error because the `copper` shapes are now empty.

Some options:
- Use `to-tuple` instead of `to-seq`. A Tuple is a Seqable!
- Pull the `min-space` calculation out and store in a variable.

Reply:
Oh, great catch, this issue has been around for a while (I did not edit this).